homework.js

```diff
@@ -2,25 +2,51 @@
 // declare a variable called fruit
 // assign your variable the value 6
 // console log your variable
-
+let fruit = 6
+console.log(fruit)
 
 // write a function called product that accepts two numbers and 
 // logs the product in the console and then
 // returns the answer when you multiply them both together
 // call that function on the numbers 4 & 689 to test it
+function product (num1, num2) {
+  let newProduct = num1 * num2;
+  console.log(newProduct);
+  returns newProduct
+}
+product (4, 689)
 
 // Problem 2
 // write a function called evens that takes an array of numbers, removes 
 // all the numbers that are odd and returns a new array with only the even numbers
 // remember that even numbers have a remainder of 0 when you divide them by 2
 // call your function on this array of numbers [4, 37, 18, 29, 52, 103, 22]
+function evens (array) {
+
```

Comment:
You'll want to use the `%` operator with a loop to check the remainder when you divide each number in the array by 2.